source/ios/AdaptiveCards/ADCIOSVisualizer/ADCIOSVisualizer/ViewController.m

```diff
@@ -218,7 +218,7 @@ - (void)update:(NSString *) jsonStr
             [self.curView removeFromSuperview];
 
         self.curView = ad;
-        if(_enableCustomRenderer){
+        //if(_enableCustomRenderer){
```

Comment:
>//if(_enableCustomRenderer){ [](start = 8, length = 28)

Don't comment out code.

Reply:
looks like i forgot to push my latest change

---
In reply to: [193241919](https://github.com/Microsoft/AdaptiveCards/pull/1547#discussion_r193241919) [](ancestors = 193241919)